src/test/java/nextstep/subway/unit/SubwayMapTest.java

```diff
@@ -78,6 +90,22 @@ void findPathOppositely() {
         assertThat(path.getStations()).containsExactlyElementsOf(Lists.newArrayList(교대역, 남부터미널역, 양재역));
     }
 
+    @DisplayName("경로중 노선의 가장높은 금액의 추가요금만 적용한다")
+    @Test
+    void 경로중_노선의_가장높은_금액의_추가요금만_적용한다() {
+        // given
+        List<Line> lines = Lists.newArrayList(신분당선, 이호선, 삼호선);
```

Comment:
자바에서 기본으로 제공하는 기능을 써보면 어떨까요?

```suggestion
        List<Line> lines = List.of(신분당선, 이호선, 삼호선);
```